src/components/fields/submit-button/submit-button.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from "@lifesg/react-design-system";
+import React, { useEffect } from "react";
+import * as Yup from "yup";
+import { useValidationSchema } from "../../../utils/hooks";
+import { IGenericFieldProps } from "../../frontend-engine";
+import { ISubmitButtonSchema } from "./types";
+
+export const SubmitButton = React.forwardRef<HTMLButtonElement, IGenericFieldProps<ISubmitButtonSchema>>(
```

Comment:
now that there is the submit button, you can remove the hardcoded submit button from frontend engine